hardware_accelerators/about-hardware-accelerators.adoc

```diff
@@ -16,7 +16,7 @@ Red{nbsp}Hat {product-title} provides support for cards and peripheral hardware
 * Data processing units (DPUs)
 
 
-image::OCP_HW_Accelerators_5.png[Supported hardware accelerators cards and peripherals]
+image::OCP_HW_Accelerators_8.png[Supported hardware accelerators cards and peripherals]
```

Comment:
@ogradyp Updated the diagram.

Reply:
Thanks @StephenJamesSmith but just to let you know that the diagram needed one more update and is now at version 9 https://issues.redhat.com/browse/DIAGRAMS-488